adr/ADR-8.md

```diff
@@ -293,6 +303,60 @@ all the known services we ask for all subjects below `$KV.SERVICES.nats.>`. A no
 
 This would mean many calls to the underlying API call for known subjects as we would not know via a typical watch about expired data.
 
+#### API Design notes
+
+The API here represents a minimum, languages can add local flavour to the API - for example one can add `PutUint64()` and `GetUint64()`
+in addition to the `Get()` and `Put()` defined here, but it's important that as far as possible all implementations strive to match
+the core API calls - `Get()`, `Put()` and to a lessor extend `Delete()` and `Purge()` and `Entry` - the rest, admin APIs, and supporting
+calls can be adjusted to be a natural fit in the language and design.  This is in line with existing efforts to harmonize 
+`Subscribe()`, `Publish()` and more.
+
+The design is based on extensive prior art in the industry. During development 20+ libraries in Go, Ruby, Python and Java were reviewed 
```

Comment:
What libraries? All etcd relative? Were cache libraries for Redis / Memchaced considered. NoSql databases?

Reply:
I looked at a bunch, for the KV stores we tend to hear from, redis a hot mess of millions of options so did not consider doing anything resembling it.

Reply:
I'm just asking to remove editorialization

Reply:
No I did not reopen anything. I capture the consensus and added details of HOW I ACTUALLY DID design this.